src/vl-map-wms-layer.js

```diff
@@ -0,0 +1,96 @@
+import {VlMapLayer} from '/src/vl-map-layer.js';
+import {
+  OlImageLayer,
+  OlImageWMSSource,
+  OlTileWMSSource,
+  OlTileLayer,
+} from '/node_modules/vl-mapactions/dist/vl-mapactions.js';
+
+/**
+ * VlMapWmsLayer
+ * @class
+ * @classdesc Deze kaartlaag staat toe om een WMS laag aan te maken.
+ *
+ * @extends VlMapLayer
+ *
+ * @property {string} data-vl-name - Attribuut bepaalt de kaartlaag naam.
+ * @property {string} data-vl-url - Attribuut bepaalt de WMS url. Verplicht.
+ * @property {string} data-vl-layers - Attribuut bepaalt de layers van de WMS. Verplicht.
+ * @property {string} [data-vl-styles=] - Attribuut bepaalt de WMS stijlen.
+ * @property {string} [data-vl-version=1.3.0] - Attribuut bepaalt de WMS versie.
+ * @property {number} [data-vl-opacity=1] - Attribuut bepaalt de WMS transparantie.
+ * @property {boolean} [data-vl-tiled=true] - Attribuut bepaalt of de WMS tiled wordt opgehaald.
+ *
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-map/releases/latest|Release notes}
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-map/issues|Issues}
+ * @see {@link https://webcomponenten.omgeving.vlaanderen.be/demo/vl-map-wms-layer.html|Demo}
+ */
+export class VlMapWmsLayer extends VlMapLayer {
+  get _url() {
+    return this.getAttribute('data-vl-url') || console.error('url not defined');
+  }
+
+  get _layers() {
+    return this.getAttribute('data-vl-layers') || console.error('layers not defined');
+  }
+
+  get _styles() {
+    return this.getAttribute('data-vl-styles') || '';
+  }
+
+  get _version() {
+    return this.getAttribute('data-vl-version') || '1.3.0';
+  }
+
+  get _opacity() {
+    return Number(this.getAttribute('data-vl-opacity') || 1);
+  }
+
+  get _tiled() {
+    return this.getAttribute('data-vl-tiled') !== 'false';
+  }
```

Comment:
inconsistentie tov van vl-map-wfs-layer en vl-map-wmts-layer in hoe hier met afwezige attributen en zo wordt omgegaan dus misschien even gelijk trekken

Reply:
Ik vond een throw 'properder' dan een console.log aangezien het niet zou mogen voorkomen (hence: throw error).

Reply:
En wat is de voorkeur?

Reply:
@pascalriquier , @wautergu en @stefanborghys ?

Reply:
als het niet kan zonder zou ik zeker error durven throwen

Reply:
ik zou ook altijd een error throwen, logs zijn verborgen en handig voor tracing. Als ik als programmeur component verkeerd gebruik wil ik duidelijk zien terugkomen en knallen, niet?

Reply:
Ik pas dit aan